### PR TITLE
Profanity filter

### PR DESCRIPTION
**Pull request by Fellylove**
# Description

This PR contains a custom profanity filter. The profanity_phrases_words directory contains two 2 CSV files, readme.MD, and the script for flagging transcription texts.  Initially, the profanity filter used profanity_filter and profanity_check libraries. However, these filters don't have an exhaustive pull of words. Also, they suffered from the "Scunthorpe effect" where words that contained profanity were flagged --which gives false positives. This function seeks to address the two problems. 

Loom video:  https://www.loom.com/share/9236075a79d747a49afedd5b65cada75

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [X ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by cryptobellum — APPROVED** [review]
Really good job on this filter! I'd suggest your README include written instructions for using the filter. Otherwise, really good work on this important component.

**Comment by Fellylove** [comment]
Thanks, @cryptobellum for pointing that out to me. I will get it fixed.

File: profanity_word_phrase_filter/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+It's important for the submitted stories to be flagged as inappropriate if they 
+contain any profanity. The stories will still be viewed by a human eye to make
+sure that the flagging is warranted. 
+The previous cohort worked on using the available libraries like profanity-filter
+and profanity_Check but they were not sufficient in flagging all words and newer
+phrases and Scuntorpe like words.
+
+Using a list of profanity phrases in (bad_phrases.csv) and profanity words in (bad_single.csv).
+bad_phrases.csv has a total of 215 phrases/expressions and bad_single.csv has a total of 1395 words.
+These words can updated according to reflect what is currently flagged as inappropriate.
```
Comment (singparvi) [inline profanity_word_phrase_filter/README.md:17]: These words can **_be_** updated according to reflect what is currently flagged as inappropriate.

File: profanity_word_phrase_filter/profanity_word_phrase_filters.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
+from json import dumps
+
+import pandas as pd
+
+# global variables
+filepath = '../profanity_word_phrase_filter/full_text.txt'
```
Comment (singparvi) [inline profanity_word_phrase_filter/profanity_word_phrase_filters.py:6]: Nicely done. Reference to the repo instead of an absolute path.

**Review by singparvi — APPROVED** [review]
Hi Sylvia this pull request was nicely done and your video explains it well.

**Pull request merged**